docker/docker_commands.txt

```diff
@@ -0,0 +1,22 @@
+// pisa net
```

Comment:
you can convert this a bash file and have commands as aliases exported to environment so they can be used more conveniently

Reply:
Not necessary, this is just a comment

Reply:
nice idea, I dont think I'll do it for now - but if I find myself using these commands a lot then I will